ReactNativeClient/lib/models/Setting.js

```diff
@@ -232,6 +232,21 @@ class Setting extends BaseModel {
 					return output;
 				},
 			},
+			layout: {
+				value: Setting.LAYOUT_ALL,
+				type: Setting.TYPE_INT,
+				public: true,
+				appTypes: ['desktop'],
+				isEnum: true,
+				label: () => _('Available Layouts'),
+				section: 'appearance',
+				options: () => ({
+					[Setting.LAYOUT_ALL]: _('Editor / Viewer / Split View'),
+					[Setting.LAYOUT_EDITOR_VIEWER]: _('Editor / Viewer'),
+					[Setting.LAYOUT_EDITOR_SPLIT]: _('Editor / Split View'),
+					[Setting.LAYOUT_VIEWER_SPLIT]: _('Viewer / Split View'),
```

Comment:
Let's make the life of translators easier by removing duplicated strings, so please change the first one to `_('%s / %s / %s', _('Editor'), _('Viewer'), _('Split View'))` and the other in a similar way.